src/App.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
-import './App.css';
+// eslint-disable-next-line sort-imports
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import SideNavbar from 'components/SideNavbar/SideNavbar';
```

Comment:
이렇게 사용하는거 뭔가 이상하지 않나요?
SideNavBar 폴더 내의 SideNavBar.js 파일명을 index.js 로 수정하면, 아래와 같이 import 할 수 있습니다.

```suggestion
import SideNavbar from 'components/SideNavbar';
```

Reply:
index 파일로 수정했습니다(완)